src/utils/processOutputInterpreters/trivy-image-output.ts

```diff
@@ -0,0 +1,112 @@
+const LineSplitter = /\r?\n/;
+const dateHeader = /^[-\d]+T[-:.\d]+Z?\s+\033\[\d+m([A-Z]+)\033\[\d+m\s+(.*)$/;
+const borderPattern = /^\+(?:-+\+){6}\s*$/;
+const internalBorderPattern = /^\+(?:(?:-+|\s+)\+){6}\s*$/;
+const rowPattern = /^(?:\|[^|]+){6}\|\s*$/;
+
+const CVEKeys = ['Package', 'VulnerabilityID', 'Severity', 'Title', 'InstalledVersion', 'FixedVersion', 'Description', 'PrimaryURL'];
+const severityRatings: Record<string, number> = {
+  LOW:      1,
+  MEDIUM:   2,
+  HIGH:     3,
+  CRITICAL: 4,
+  UNKNOWN:  5,
+};
+const MaxSeverityRating = Math.max(...Object.values(severityRatings));
+
+type finalVulType = Record<string, string>;
+
+export default class TrivyScanImageOutputCuller {
+  prelimLines: Array<string>;
+  JSONLines: Array<string>;
+  inJSON = false;
+
+  constructor() {
+    this.prelimLines = [];
+    this.JSONLines = [];
+  }
+
+  getRating(key: string) {
+    return key in severityRatings ? severityRatings[key] : MaxSeverityRating;
+  }
+
+  fixLines(lines: Array<string>) {
+    // "key": "value with an escaped \' single quote isn't valid json"
+    return lines.map(line => line.replace(/\\'/g, "'"));
+  }
+
+  addData(data: string): void {
+    if (this.inJSON) {
+      this.JSONLines.push(data.replace(/\\'/g, "'"));
```

Comment:
```suggestion
      this.JSONLines.push(...fixLines([data]));
```
Just trying out alternatives :)

Can we assume that `data` here is exactly one full line?

Reply:
That would be covered by #443